python/pyspark/sql/tests/test_udtf.py

```diff
@@ -397,6 +397,198 @@ def test_udtf(a: int):
         with self.assertRaisesRegex(TypeError, err_msg):
             udtf(test_udtf, returnType="a: int")
 
+    def test_udtf_with_table_argument_query(self):
+        class TestUDTF:
+            def eval(self, row: Row):
```

Comment:
To use the TABLE argument, the signature of the UDTF must be a row right? 
Maybe we should mention in this the udtf doc.

Reply:
Yes, `Row` object will be passed when the TABLE argument is used.